Overview.src.html

```diff
@@ -2087,6 +2094,11 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
      "<code title>manual</code>" and <var title>response</var>'s
      <span title=concept-response-type>type</span> is "<code title>opaqueredirect</code>".
     </ul>
+
+   <li>
+     <p>Execute <a href=https://w3c.github.io/webappsec-csp/set-response-policy-list>Set
+     <var title>response</var>'s policy list</a> on <var title>response</var>.
+     <span data-anolis-ref>CSP</span>
```

Comment:
I've taken a stab at this in the current Fetch patch, and poked at the CSP side in https://github.com/w3c/webappsec-csp/commit/0545ae08d6c74788f006e07b2b991bbf925c9691. WDYT?